# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cmake/hotfix-casesensitivity.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Since cmake-4, the checks for case insensitivity open many bindirs and other base
folders, that lead to the wrong detection of dependencies, so we disable that.

--- cmake-4.1.1/Source/cmSystemTools.cxx.vanilla	2025-09-15 23:30:37.462915340 +0200
+++ cmake-4.1.1/Source/cmSystemTools.cxx	2025-09-15 23:30:43.857735347 +0200
@@ -4189,6 +4189,7 @@
 #if defined(_WIN32) || defined(__APPLE__)
   return DirCase::Insensitive;
 #elif defined(__linux__)
+  return DirCase::Sensitive;
   int fd = open(dir.c_str(), O_RDONLY);
   if (fd == -1) {
     // cannot open dir but it exists, assume dir is case sensitive.
